hydromt/cli/main.py

```diff
@@ -255,6 +256,7 @@ def update(
     opt,
     data,
     dd,
+    fo,
```

Comment:
why `fo` and not `fc` though I wonder?

Reply:
It means force overwrite and we define that flag for the build method (in that case you also overwrite files but in build mode you did not try to open them first), so I thought I could re-use the flag. If it bothers you you could also rename forceful_cleanup to force_overwrite as well for consistency?

Reply:
ahhh that makes sense. Yeah I think it's better to stay consistent, but I don't mind doing the renaming

Reply:
Done :)